ReactNativeClient/lib/services/InteropService_Exporter_Json.js

```diff
@@ -0,0 +1,80 @@
+const InteropService_Exporter_Base = require('lib/services/InteropService_Exporter_Base');
+const { basename, filename } = require('lib/path-utils.js');
+const { shim } = require('lib/shim');
+
+class InteropService_Exporter_Json extends InteropService_Exporter_Base {
+
+	async init(destDir) {
+		this.destDir_ = destDir;
+		this.resourceDir_ = destDir ? destDir + '/resources' : null;
+
+		await shim.fsDriver().mkdir(this.destDir_);
+		await shim.fsDriver().mkdir(this.resourceDir_);
+	}
+
+	async processItem(ItemClass, item) {
+		const obj = await this.buildPlainObjectForJson_(ItemClass, item);
+		const fileName = this.getNameWithDifferentExtension_(ItemClass, item, ".json");
```

Comment:
Instead of having a specialised function here, I think it would be better to change the systemPath method to this:

```
	static systemPath(itemOrId, extension = null) {
		if (extension === null) extension = 'md'
		if (typeof itemOrId === 'string') return itemOrId + '.' + extension;
		return itemOrId.id + '.' + extension;
	}
```

Then here you can simply do `const filename = ItemClass.systemPath(item, 'json');`

Reply:
Agreed, done.